core/templates/dev/head/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -31,7 +44,64 @@
     </div>
     <div ng-cloak class="navbar-header float-right">
       <ul class="nav oppia-navbar-nav oppia-navbar-profile-admin">
-        <ul class="nav oppia-navbar-tabs-admin">
+        <ul class="nav oppia-navbar-tabs-admin hidden-icon" uib-dropdown>
```

Comment:
Can you find a way to parameterize this so that not so much code is repeated? E.g. you can have a list containing the names and click targets of the tabs, then use ng-repeat. That can be used for the main wide-screen menu as well.

Otherwise this code is starting to get a bit too bulky.

Reply:
Okay, will look into it.